### DISCUSSION TITLE
integral

### DISCUSSION THREAD

**Discussion by thistlillo** [artisticat1/obsidian-latex-suite#308] [Q&A] [answered]
When I type `\int`, the text gets automatically replaced by `\in t`.

Looking here:
https://github.com/artisticat1/obsidian-latex-suite/blob/main/src/default_snippets.js

I cannot see which one is the snippet responsible for the replacement.

What line should I remove?

**Comment by superle3** [comment, marked as answer]
(a bit late)
See https://github.com/artisticat1/obsidian-latex-suite/blob/1b010e18a58ee1a0d50f2520908533a7cc309b89/src/default_snippets.js#L162-L163
`in` is probably in one of the symbols.

Probably better to create a snippet with higher priority for int alone as the space helps a lot for the rest.
